financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/ShapedIcon.kt

```diff
@@ -29,14 +28,19 @@ private val iconSize = 20.dp
 @Composable
 internal fun ShapedIcon(
     painter: Painter,
+    modifier: Modifier = Modifier,
+    iconSize: IconSize = IconSize.Medium,
```

Comment:
Adapted our icon to support sizes, matching web - https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/5573b9b445dfaac104225d6e7969035f3cc26701/src/linkedAccounts/shared-ui/v3/Icon/index.tsx#L47